microsoft/testsuites/network/sriov.py

```diff
@@ -664,6 +666,48 @@ def verify_sriov_ethtool_offload_setting(self, environment: Environment) -> None
         iperf_log = dest_cat.read(client_iperf3_log, sudo=True, force_run=True)
         assert_that(iperf_log).does_not_contain("error")
 
+    @TestCaseMetadata(
+        description="""
+        This test case verifies that irq rebalance is running.
+        An issue was previously seen in irqbalance 1.8.0-1build1 on Ubuntu.
```

Comment:
Add what is expected, when it is unexpected, what may happen, what we should do.